config/settings/base.py

```diff
@@ -0,0 +1,127 @@
+import os
+from dotenv import load_dotenv
+
+
+def reduce_path(file_name, times=3):
+    """Return directory path"""
+    result = os.path.realpath(file_name)
+    for _ in range(times):
+        result = os.path.dirname(result)
+    return result
+
+
+def rel_path(*x):
+    """Handling file paths"""
+    return os.path.abspath(os.path.join(BASE_DIR, *x))
+
+
+# Project directory
+BASE_DIR = reduce_path(__file__)
+
+# .env file
+dotenv_path = rel_path('.env')
+
+# load settings from .env file
+load_dotenv(dotenv_path)
+
+SECRET_KEY = os.getenv('SECRET_KEY')
+
+DJANGO_APPS = [
+    'django.contrib.admin',
+    'django.contrib.auth',
+    'django.contrib.contenttypes',
+    'django.contrib.sessions',
+    'django.contrib.messages',
+    'django.contrib.staticfiles',
+]
+
```

Comment:
Remove

Reply:
Why I should remove it? You wrote me like that: 
"I would split INSTALLED_APPS into DJANGO_APPS, THIRD_PARTY_APPS and LOCAL_APPS and then make:"

Reply:
You should remove empty lines :)
Sorry for the confusion!